### ISSUE TITLE
how to use  ant.design UI in my plugin ?

### ISSUE THREAD

**Issue by pnl107** [strapi/strapi#3019]
**Informations**
- **Node.js version**:  v10.15.3 
- **NPM version**:   v6.4.1 
- **Strapi version**:3.0.0-alpha.25.2
- **Database**: 
- **Operating system**:  Linux 
- **(Optional) Link to your Project**: 

after : **npm install antd** and use component in antd then "fail"

https://ant.design/docs/react/introduce

**What is the current behavior?**

![image](https://user-images.githubusercontent.com/39798640/54877068-5611d100-4e4c-11e9-817a-bfd9afd1e0a4.png)

**Steps to reproduce the problem**

**What is the expected behavior?**

**Suggested solutions**

**Comment by soupette** [comment]
Hi @pnl107 you need to install the ant design in the `strapi-helper-plugin` and then modify the webpack configuration so it is available in your plugin.

Please refer the dev, prod and dll configurations


https://github.com/strapi/strapi/blob/a42c2a483ca6e35a25c7d214bedd2e0cefc48dbb/packages/strapi-helper-plugin/lib/internals/webpack/webpack.dev.babel.js#L226-L232

https://github.com/strapi/strapi/blob/a42c2a483ca6e35a25c7d214bedd2e0cefc48dbb/packages/strapi-helper-plugin/lib/internals/webpack/webpack.dll.babel.js#L148-L154

https://github.com/strapi/strapi/blob/a42c2a483ca6e35a25c7d214bedd2e0cefc48dbb/packages/strapi-helper-plugin/lib/internals/webpack/webpack.prod.babel.js#L243-L249

Basically all you need to do is install your dependency in the strapi-helper-plugin, add it to the webpack configuration then you can import it in your plugin.

**Comment by pnl107** [comment]
<img width="1098" alt="Screen Shot 2019-03-27 at 3 42 43 PM" src="https://user-images.githubusercontent.com/39798640/55062788-3d522700-50a9-11e9-8c48-ef1cf94a7e17.png">
<img width="1395" alt="Screen Shot 2019-03-27 at 3 42 59 PM" src="https://user-images.githubusercontent.com/39798640/55062789-3deabd80-50a9-11e9-8a72-cdebef0be895.png">
<img width="1439" alt="Screen Shot 2019-03-27 at 3 58 04 PM" src="https://user-images.githubusercontent.com/39798640/55062790-3deabd80-50a9-11e9-9e92-13d906391d85.png">
it not work :(
